bgpd/bgp_vty.c

```diff
@@ -7508,16 +7511,25 @@ DEFUN (show_bgp_vrfs,
 			json_object_string_add(
 				json_vrf, "rmac",
 				prefix_mac2str(&bgp->rmac, buf, sizeof(buf)));
+			json_object_string_add(json_vrf, "interface",
```

Comment:
svi-interface instead of interface ?

Reply:
"interface" is the consistent keyword across all json outputs.